### DISCUSSION TITLE
I can't find a proper answer for plugin Tflite uses a deprecated version of Android embedding

### DISCUSSION THREAD

**Discussion by LoftyDroid** [community/community#53559] [Mobile]
### Select Topic Area

Question

### Body

I have updated and tflite_flutter but still the gradle is not building

```
tflite: ^1.1.2
tflite_flutter: ^0.9.1
```
The following this the error I'm getting:

The plugin `tflite` uses a deprecated version of the Android embedding.

To avoid unexpected runtime failures, or future build failures, try to see if this plugin supports the Android V2 embedding. Otherwise, consider removing it since a future release of Flutter will remove these deprecated APIs.

If you are plugin author, take a look at the docs for migrating the plugin to the V2 embedding: https://flutter.dev/go/android-plugin-migration.

main.dart:1

FAILURE: Build failed with an exception.

* Where:

Build file 'C:\Users\Dell\AppData\Local\Pub\Cache\hosted\pub.dev\tflite-1.1.2\android\build.gradle' line: 24

I have gone through the docs, but I don't still understand what I need to do, Can someone please give a step by steps explanation regarding this issue,

Here is the Github repo on which I'm working: https://github.com/LoftyDroid/Plant-Disease-Detection-Flutter.git

you can try running this on your own PC to check if the solution is working or not and I don't mind contributions if you can.

**Comment by taki-on** [comment]
Hello! Your question seems related to Tensor Flow Lite and/or Flutter. This forum is about discussing features and bugs in GitHub Mobile apps. You might want to find the right forum to ask this question. Thanks!
